### PR TITLE
OE0-32: added new extensions, update existing and remove not used in profile

### PR DESCRIPTION
**Pull request by sniedzielski**
TICKET: https://openimis.atlassian.net/browse/OE0-32

- Adding new extensions
- updating contact with adding displat/code and fixing URL. 
- Removing extensions not used like ValidityFrom, Poverty etc
- uppdate crating build_imis_extension and build_imis_address

related PR - https://github.com/openimis/openimis-be-api_fhir_r4_py/pull/31

### CODE REVIEW

File: api_fhir_r4/mapping/patientMapping.py [hunk 1]
```diff
@@ -11,3 +11,36 @@ class RelationshipMapping(object):
         "7": "Others",
         "8": "Spouse",
     }
+
+
+class EducationLevelMapping(object):
+
+    education_level = {
```
Comment (dragos-dobre) [inline api_fhir_r4/mapping/patientMapping.py:18]: Get the list from DB.

File: api_fhir_r4/mapping/patientMapping.py [hunk 2]
```diff
@@ -11,3 +11,36 @@ class RelationshipMapping(object):
...
+        "4": "University",
+        "5": "Postgraduate studies",
+        "6": "PHD",
+        "7": "Other",
+    }
+
+
+class PatientProfessionMapping(object):
+
+    patient_profession = {
```
Comment (dragos-dobre) [inline api_fhir_r4/mapping/patientMapping.py:31]: Get the list from DB.

File: api_fhir_r4/mapping/patientMapping.py [hunk 3]
```diff
@@ -11,3 +11,36 @@ class RelationshipMapping(object):
...
+        "1": "Housewife",
+        "2": "Employee",
+        "3": "Self Employee",
+        "4": "Other",
+    }
+
+
+class IdentificationTypeMapping(object):
+
+    identification_type = {
```
Comment (dragos-dobre) [inline api_fhir_r4/mapping/patientMapping.py:41]: Get the list from DB.

**Review by dragos-dobre — CHANGES_REQUESTED** [review]

File: api_fhir_r4/mapping/patientMapping.py (hunk 1 above)
Comment (sniedzielski) [reply api_fhir_r4/mapping/patientMapping.py:20]: fixed

File: api_fhir_r4/mapping/patientMapping.py (hunk 2 above)
Comment (sniedzielski) [reply api_fhir_r4/mapping/patientMapping.py:27]: fixed

File: api_fhir_r4/mapping/patientMapping.py (hunk 3 above)
Comment (sniedzielski) [reply api_fhir_r4/mapping/patientMapping.py:34]: fixed

File: api_fhir_r4/mapping/patientMapping.py [hunk 4]
```diff
@@ -1,3 +1,5 @@
+from insuree.models import Profession, Education, IdentificationType
+
 
 class RelationshipMapping(object):
```
Comment (dragos-dobre) [inline api_fhir_r4/mapping/patientMapping.py:5]: Get the list from DB.

**Review by dragos-dobre — CHANGES_REQUESTED** [review]

File: api_fhir_r4/mapping/patientMapping.py (hunk 4 above)
Comment (sniedzielski) [reply api_fhir_r4/mapping/patientMapping.py:5]: ok

Comment (sniedzielski) [reply api_fhir_r4/mapping/patientMapping.py:5]: fixed

**Review by dragos-dobre — APPROVED** [review]

**Pull request merged**